bot/cogs/tags.py

```diff
@@ -92,9 +92,8 @@ def _get_suggestions(self, tag_name: str, thresholds: Optional[List[int]] = None
         """Show all known tags, a single tag, or run a subcommand."""
         await ctx.invoke(self.get_command, tag_name=tag_name)
 
-    @tags_group.command(name='get', aliases=('show', 'g'))
-    async def get_command(self, ctx: Context, *, tag_name: TagNameConverter = None) -> None:
-        """Get a specified tag, or a list of all tags if no tag is specified."""
+    async def _get_command(self, ctx: Context, tag_name: TagNameConverter = None) -> None:
```

Comment:
Considering it is accessed elsewhere (i.e. in `error_handler`), it doesn't seem appropriate to make this function "private" by prefixing its name with an underscore. And since it won't be private, it should have a docstring (explain what it returns too).

Reply:
Alright, right away

Reply:
This is the command itself, are you referring to the explicit one we separated?

Reply:
The outer function (the one that is being called inside the command) is accessible within the error handler, I applied these changes to the outer function

Reply:
Was referring to `_get_command` to be clear.